src/Routing/RouteSubscriber.php

```diff
@@ -66,6 +67,23 @@ public function routes() {
       ));
     }
 
+    /** @var \Drupal\sms\Entity\SmsGatewayInterface $gateway */
+    foreach (SmsGateway::loadMultiple() as $gateway) {
+      if (!$gateway->supportsReportsPush()) {
+        continue;
+      }
+
+      $id = $gateway->id();
+      $path = $gateway->getPushReportPath();
+      if (Unicode::strlen($path) >= 2 && Unicode::substr($path, 0, 1) == '/') {
```

Comment:
Yeh I looked. Search the code base for ` !== '/'`. Path module and widgets have it in their validators.

Path services dont seem to be much help.